visual-tests/scripts/visual-tests-build.mjs

```diff
@@ -0,0 +1,72 @@
+import path from 'path';
+import execa from 'execa';
+import fse from 'fs-extra';
+
+// Main wrapper should be defined as a first one -
+// it will be used to generate golden screenshots,
+// rest of wrappers will be compared to it.
+const wrappers = ['js', 'angular-13', 'react'];
+// const wrappers = ['js', 'angular-13', 'react', 'vue'];
+const baseBranch = 'develop';
+const currentBranchName = (await execa.command('git rev-parse --abbrev-ref HEAD', { silent: true })).stdout;
+const dirs = {
+  init: process.cwd(),
+  examples: '../examples/next/visual-tests',
+  codeToRun: 'demo',
+};
+
+// const HOTversion = '12.2.0';
```

Comment:
Is this commented-out line needed?

Reply:
not anymore - it was needed to test external URLs and looks like we won't back to this solution. Removed